blaze-client/src/main/scala/org/http4s/client/blaze/BlazeClientConfig.scala

```diff
@@ -49,7 +49,7 @@ final case class BlazeClientConfig( // HTTP properties
     maxConnectionsPerRequestKey: RequestKey => Int,
     // security options
     sslContext: Option[SSLContext],
-    @deprecatedName('endpointAuthentication) checkEndpointIdentification: Boolean,
+    @deprecatedName(Symbol("endpointAuthentication")) checkEndpointIdentification: Boolean,
```

Comment:
The symbol version of `@deprecatedName` is also deprecated.  I've started sadly phasing this out of various projects.  I think it's mostly useful in stdlib and non-crossbuilt code.